src/main/java/core/basesyntax/service/strategy/strategyimpl/BalanceOperation.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.strategy.strategyimpl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.service.strategy.OperationHandler;
+import java.util.HashMap;
+
+public class BalanceOperation implements OperationHandler {
+    private final StorageDao actionDB;
```

Comment:
this is confusing, same for all handlers
```suggestion
    private final StorageDao storageDao;
```